### PR TITLE
Replaced 'Profiles' queries with 'User' queries.

### PR DESCRIPTION
**Pull request by porckhop**
I have already tested all the changes and it all works fine. The only thing left after this is to update the 'User' validation data in MongoDB.

### CODE REVIEW

File: backend/Social/SocialAPI.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def profiles():
 def users():
     userIdList = request.args.getlist('userID')
     try:
-        data = db.Profiles.find({"userID": {'$in': userIdList}}, {"_id": 0})
+        data = db.User.find({"userID": {'$in': userIdList}}, {"_id": 0})
```
Comment (angnobel) [inline backend/Social/SocialAPI.py:75]: Please dont return the passwordHash for this

/profile?userID=A0223921M

File: backend/Social/SocialAPI.py [hunk 2]
```diff
@@ -88,7 +88,7 @@ def users():
 @cross_origin(supports_credentials=True)
 def getUserProfile(userID):
     try:
-        data = db.Profiles.find({"userID": userID}, {"_id": 0})
+        data = db.User.find({"userID": userID}, {"_id": 0})
```
Comment (angnobel) [inline backend/Social/SocialAPI.py:91]: Please dont return the passwordHash for this

File: backend/Social/SocialAPI.py [hunk 3]
```diff
@@ -164,7 +164,7 @@ def getUserDetails(userID):
             }
             },
             {'$lookup': {
-                'from': 'Profiles',
+                'from': 'User',
```
Comment (angnobel) [inline backend/Social/SocialAPI.py:167]: Please dont return the passwordHash for this

**Review by angnobel — CHANGES_REQUESTED** [review]
Generally okay, please dont return the pasword hash

**Comment by porckhop** [comment]
I have prevented passwordHash from being projected in all User queries instances. I thought this would be good, in case the code gets changed and the passwordHash is accidently projected in the future.

**Review by angnobel — APPROVED** [review]
Merge after reintegrate

**Pull request merged**